packages/calcite-components/src/components/card-group/card-group.tsx

```diff
@@ -215,7 +215,7 @@ export class CardGroup implements InteractiveComponent, LoadableComponent {
       });
     }
 
-    this.selectedItems = this.items.filter((el) => el.selected);
+    this.selectedItems = this.items.filter((el) => el.selected || el.hasAttribute("selected"));
```

Comment:
Before, card-group and card where in the same lazy loading bundle so by the time card-group executes this function, the `card` constructor is definitely loaded and attributes are reflected to properties.
With Lumina, we do not put related components into bundle yet, so `card` is not guaranteed to be loaded by now. Until `card` is loaded, attribute is not reflected to a property so we must check both.

The other option is to call this `setSelectedItems()` again in `loaded()` as `card` is guaranteed to be loaded by then.

Reply:
edit:
undoing this change for now because reflected attributes are not updated until re-render, so this change is causing `this.selectedItems` to be computed based on old value of `.selected` prop